### PR TITLE
feat: added product to wishlist and saved locally

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
files to check : 
src/context/WishListContext.js
src/pages/wishlist-page/Wishlist.js
src/components/product-card/ProductCard.jsx

### CODE REVIEW

File: src/context/WishListContext.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext,  useEffect,  useState } from "react";
+
+const WishListContext = createContext(null);
+
+const WishListProvider = ({children}) => {
+    
+    const [ wishlist, setWishList ] = useState(JSON.parse(localStorage.getItem ("wishlist")));
```
Comment (rutu14) [inline src/context/WishListContext.js:7]: Suggestion: Can try not storing in local storage.

**Review by rutu14 — APPROVED** [review]
The code is good to merge.

File: src/context/WishListContext.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/context/WishListContext.js:7]: i will try later

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Comment by santhoshpemmaka** [comment]
Code looking Good. 👍  Good to merge 💯

**Review by santhoshpemmaka — APPROVED** [review]

**Pull request merged**